lib/pdf_fill/hash_converter.rb

```diff
@@ -125,19 +127,35 @@ def check_for_overflow(arr, pdftk_keys)
       false
     end
 
+    def handle_overflow_and_label_all(form_data, pdftk_keys)
+      form_data.each_with_index do |item, idx|
+        item.each do |k, v|
+          text = overflow?(pdftk_keys[k], v) ? EXTRAS_TEXT : v
+
+          set_value(text, pdftk_keys[k], idx, true) if pdftk_keys[k].is_a?(Hash)
+        end
+      end
+    end
```

Comment:
Can you add a test for this in the hash_converter spec? When i run the file locally on your branch, I see the method isn't getting tested:
![image](https://github.com/user-attachments/assets/f9655a87-411c-4ab1-9567-4eb91d420ee4)

Reply:
A lot better. thanks! 
![image](https://github.com/user-attachments/assets/f2eb2625-107d-4da7-ac0d-53aad7dba71c)